quantum/process_keycode/process_autocorrect.c

```diff
@@ -19,6 +19,30 @@
 #    include "autocorrect_data_default.h"
 #endif
 
+#ifndef AUTOCORRECT_LINK_BYTE_COUNT
+#    define AUTOCORRECT_LINK_BYTE_COUNT 2
+#endif
+
+#if AUTOCORRECT_LINK_BYTE_COUNT < 2 || AUTOCORRECT_LINK_BYTE_COUNT > 3
+#    error "AUTOCORRECT_LINK_BYTE_COUNT must be either 2 or 3 bytes."
+#endif
```

Comment:
is there a specific reason to limit to 3 bytes here, rather than going to straight up 4 and using the full uint32 for the state?